### ISSUE TITLE
Make generator conda-installable

### ISSUE THREAD

**Issue by ijstokes** [Anaconda-Server/generator-nbextension#1]
I would like to be able to do:

```bash
source activate root
conda install -c anaconda-notebook generator-nbextension
extension_template [--options]
```

And if that is complicated, then the last line could instead be:

```bash
yo nbextension
```

but I would prefer not to have to expose the user to `yo` or any other `node`, `yeoman` or `nodejs` intermediate tooling.

**Comment by malev** [comment]
@ijstokes if you are ok with:
`conda install generator-nbextension -c anaconda-notebook -c javascript` and `yo nbextension` then we are good :smile:

**Comment by ijstokes** [comment]
@malev that is fine with me.  The only surprising bit (but I've never used "yeoman" before) was that I needed to execute the command in a new directory that I created first.  When I tested it I was surprised that it dumped a bunch of files in my *current* directory.  I expected they would all be encapsulated in a single sub-directory.

Anyway, this is OK for now at least (and maybe that behavior is expected), and the extension that was created worked nicely.  So I'm calling this done.
